<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_charSheet</class>
 <widget class="QDialog" name="Dialog_charSheet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../../home/martin/proj/pq/progress-quest/crossed_swords_sm.gif</normaloff>../../../../../../home/martin/proj/pq/progress-quest/crossed_swords_sm.gif</iconset>
  </property>
  <property name="toolTip">
   <string extracomment="Create specific character"/>
  </property>
  <widget class="QGroupBox" name="gbx_charOV">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>0</y>
     <width>171</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Character Overview</string>
   </property>
   <widget class="QLineEdit" name="lne_name">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="Enter the name of the character"/>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_race">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="Select race"/>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_class">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="Select class"/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_name">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_race">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Race</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_class">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Class</string>
    </property>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="bbx_charDialog">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>210</y>
     <width>176</width>
     <height>27</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="gbx_stats">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>171</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>Stats</string>
   </property>
   <widget class="QSpinBox" name="sbx_str">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_str">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="Strength"/>
    </property>
    <property name="text">
     <string>STR</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbx_int">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_int">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="Intellegence"/>
    </property>
    <property name="text">
     <string>INT</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbx_dex">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_dex">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="Dexterity"/>
    </property>
    <property name="text">
     <string>DEX</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_wis">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="Wisdom"/>
    </property>
    <property name="text">
     <string>WIS</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbx_wis">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_con">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="Constitution"/>
    </property>
    <property name="text">
     <string>CON</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbx_con">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbx_chr">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_chr">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="Charisma"/>
    </property>
    <property name="text">
     <string>CHR</string>
    </property>
   </widget>
  </widget>
  <widget class="QLCDNumber" name="lcd_spare">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string extracomment="Points that can be applied to stats"/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_lcd">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Spendable</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tbn_lcd">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string extracomment="Randomize point distribution"/>
   </property>
   <property name="text">
    <string>-R-</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
